src/app/merge-diff/merge-diff-container/merge-diff-container.component.html

```diff
@@ -16,4 +16,72 @@
 
 SPDX-License-Identifier: Apache-2.0
 -->
-<p>merge-diff-container works!</p>
+<div class="loading-spinner" *ngIf="!loaded">
+  <div>
+    <i class="fas fa-sync-alt fa-spin"></i>
+  </div>
+</div>
+<div  *ngIf="loaded">
+  <div class="heading-container">
```

Comment:
Using the `heading-container` CSS class means the right-side controls, such as the "Commit" button, don't look in line with the rest of the page container:

![image](https://user-images.githubusercontent.com/3219480/122794983-be162280-d2b4-11eb-9709-7eec8ca3c76a.png)

Would it look better to line everything up by using the `container` class for all block elements?